project/scripts/genDocs

```diff
@@ -57,10 +57,10 @@ if [ "$1" = "test" ]; then
 
     # add all contents of $PWD to commit
     git add -A
-    git commit -m "Update gh-pages site for $GIT_HEAD"
+    git commit -m "Update gh-pages site for $GIT_HEAD" || true
```

Comment:
Currently trying to get something else to work, will log here instead later instead of silently failing